cloudbuild.yaml

```diff
@@ -20,6 +20,11 @@ steps:
         '--tag=gcr.io/oak-ci/oak:latest',
         '.',
       ]
+    env:
+      [
+        "DOCKER_BUILDKIT=1",
```

Comment:
(nit) we seem to be using single-quoted string elsewhere, so perhaps worth sticking with that convention (it may even be enforced by some linters already, not sure)